### ISSUE TITLE
Doesnt work on MacOS Sierra

### ISSUE THREAD

**Issue by bkerensa** [unetbootin/unetbootin#80]

**Comment by Aeirex** [comment]
It works for me.

**Comment by idarek** [comment]
Where we can download this "Public Release Candidate"?

**Comment by Aeirex** [comment]
It is the last Public Beta, macOS is getting the update in a few days. No point upgrading now. But you have to be a developer/be apart of the beta program. (Being a developer doesn't require anything.)
https://beta.apple.com/sp/betaprogram/

**Comment by idarek** [comment]
It is likely that GM (Golden Master) is final version of macOS same as it was with iOS 10, so is worth to try thing now auto be ready on lunch.

**Comment by Aeirex** [comment]
Well it works perfectly fine in macOS Sierra. I made a bootup USB using it just fine.

**Comment by idarek** [comment]
Strange, as I just run it and it didn't see any USB devices.

**Comment by Aeirex** [comment]
It may be a bug with something else, they show on my MacBook Pro and my Hackintosh computer.

**Comment by scano** [comment]
Works for me on macOS Sierra final release.
Format the USB in FAT with GUID partition map.

**Comment by idarek** [comment]
Yep, checked, and GUID is the case :)... nothing to do with macOS, Thanks :)

**Comment by MatzFan** [comment]
Should this issue be closed if the issue is not with macOS?

**Comment by idarek** [comment]
Yes, it's depend how pendrive is formatted etc
